iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergAcidUtil.java

```diff
@@ -243,14 +247,49 @@ public static long getDeleteFilePosition(Record rec) {
     return rec.get(DELETE_FILE_META_COLS.get(MetadataColumns.ROW_POSITION), Long.class);
   }
 
+  private static long hashObjectArray(Object[] values) {
+    Hasher hasher = Hashing.murmur3_128().newHasher();
+
+    for (Object val : values) {
+      if (val == null) {
+        // Unique constant for null
+        hasher.putInt(0xDEADBEEF);
```

Comment:
`0xDEADBEEF`
What is this value?

Reply:
> The number 0xDEADBEEF is a hexadecimal literal often used in programming as a sentinel, debug marker, or placeholder value. It is interpreted as -559038737 in Java

This is a unique constant for null to avoid 0 and null contribute the same hashing value.